src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -4840,6 +4840,325 @@ void M()
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedGraph, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void UsingDeclaration_Flow_12()
+        {
+            string source = @"
+#pragma  warning disable CS0815, CS0219, CS0164
+class P : System.IDisposable
+{
+    public void Dispose()
+    {
+    }
+
+    void M()
+    /*<bind>*/{
+        int a = 0;
+        int b = 1;
+label1:
```

Comment:
>label1 [](start = 0, length = 6)

I think it would be good to test scenarios when there is a conditional goto to the label:
- From above
- From below
- From above and below

Reply:
For new tests limit declarations to one using declaration per scenario.

---
In reply to: [338766750](https://github.com/dotnet/roslyn/pull/39451#discussion_r338766750) [](ancestors = 338766750)